hd-docs/others/project-structure.md

```
@@ -0,0 +1,126 @@
+- **Prisma ORM**: Type-safe database access with automatic migrations
+- **SQLite**: Default database for simplicity (production supports PostgreSQL, MySQL)
+- **Migration Management**: Automated via `yarn prisma:setup`
+
+## Environment Configuration
+
+The project uses environment-specific configuration:
+
+```javascript
+// Dynamic environment loading
+process.env.NODE_ENV === "development"
+  ? require("dotenv").config({ path: `.env.${process.env.NODE_ENV}` })
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)